seed_identity_store/settings.py

```diff
@@ -212,3 +213,27 @@
 
 METRICS_URL = os.environ.get("METRICS_URL", None)
 METRICS_AUTH_TOKEN = os.environ.get("METRICS_AUTH_TOKEN", "REPLACEME")
+
+
+from seed_papertrail.config import handler, formatter  # noqa
+
+PAPERTRAIL = os.environ.get('PAPERTRAIL')
+if PAPERTRAIL:
+    PAPERTRAIL_HOST, _, PAPERTRAIL_PORT = PAPERTRAIL.partition(':')
+    LOGGING = {
```

Comment:
@jpaidoussi it looks like raven works even if `LOGGING = {}` so it's insert the fields it needs somewhere itself or is hooking in at a lower level using Python's logging module directly.